### ISSUE TITLE
HSLuv support

### ISSUE THREAD

**Issue by MartinKavik** [rebo/seed_style_preview#7]
https://www.hsluv.org/

New features:
```rust
CssColor::Hsluv(260.4, 80.5, 37.9) 
hsluv(260, 80, 38)
```
- H: The hue of the color. Should be between 0 and 360 degrees, inclusive. f64.
  - [Red - Green (120) - Blue (240) - Red] 
- S: The saturation of the color. Must be between 0% and 100%, inclusive. f64.
- L: The lightness of the color. Must be between 0% and 100%, inclusive. f64.

Should it panic or return `Err` if a value is out of range?

Crate for converting HSLuv to HEX and RGB - https://crates.io/crates/rust-hsluv.

Do we want to implement also `hslauv` (inspired by CSS [hsla](https://www.w3schools.com/cssref/func_hsla.asp))?
```rust
CssColor::Hslauv(260.4, 80.5, 37.9, 0.3) 
hslauv(123, 45, 67, 1)
```
- A: Defines the opacity as a number between 0.0 (fully transparent) and 1.0 (fully opaque).

**Comment by rebo** [comment]
hIsa implemented in master as a `CssColor` variant.

```rust
CssColor::Hsla(f64,f64,f64,f64)
```

including associated helper function `hsla()`.

`hsluv()` and `hsluva()` helper functions implemented but they work through conversion to `CssColor::Rgba`.  

I didn't go the whole hog and add variants for these as need some further clarification on the ergonomics on these functions.  Also the hsluv crate is large 1.1mb, it doesn't seem to a fact pkg size in my test.  I want to do some testing first before I tie it completely into everything with additional CssColor variants.

Im closing this and adding an issue on helper ergonomics.

**Comment by rebo** [comment]
closing as implemented in caf8cde
